test/vector3.cpp

```diff
@@ -27,6 +27,19 @@ int main()
     TEST(isEqual(a.unit(), a.normalized()));
     TEST(isEqual(a*2.0, Vector3f(2, 0, 0)));
 
+    Vector2f g2(1,3);
+    Vector3f g3(7, 11, 17);
+    g3.xy() = g2;
```

Comment:
@bresch how does this look?

Reply:
That's awesome! Thanks a lot. After that gets available in the firmware, I'll make a pass to get rid of the 1-by-1 assignments for xy axes.